docs/docs/sql/fp.md

```diff
@@ -1,5 +1,9 @@
-Floating-point arithmetic
-=========================
+---
```

Comment:
this file should probably be deleted.

Reply:
I'll leave it up to you to organize it I just made sure it has all the original content from your docs.